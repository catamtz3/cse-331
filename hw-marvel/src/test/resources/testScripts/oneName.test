LoadGraph oneName oneName.csv

FindPath oneName CAT CAT
