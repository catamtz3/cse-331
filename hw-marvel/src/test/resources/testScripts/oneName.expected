loaded graph oneName

path from CAT to CAT:
